influence_benchmark/config/experiment_configs/EI_test.yaml

```diff
@@ -14,7 +14,7 @@ final_reward: false
 num_gen_trajs_per_initial_state: 1
 top_n_trajs_per_initial_state: 1
 iterations: 1
-pm_length_penalty: null
+pm_length_penalty: 5.0e-4
```

Comment:
I think this is quite a high length penalty? I usually do a factor 10 less. Not sure what the optimum is

Reply:
I don't care, this is just for testing that it doesn't crash and burn

Reply:
the _up test is more important, and nowadays the real runs are what actually matter. These smaller tests are just to catch syntax errors